lib/manageiq/environment.rb

```diff
@@ -24,6 +24,8 @@ def self.manageiq_plugin_update(plugin_root = nil)
       create_database_user if ENV["CI"]
 
       setup_test_environment(:task_prefix => 'app:', :root => plugin_root)
+
+      prepare_codeclimate_test_reporter if ENV["CI"]
```

Comment:
@agrare Only thing is that once we merge this, then all repos will start using the new test reporter...we may have to blast out changes so that the corresponding after_script is in place

Reply:
Okay so you want to blast out the after script to all the repos then merge this?